client/src/pages/Signup.js

```diff
@@ -64,6 +58,25 @@ const Signup = () => {
   };
 
   const submit = () => {
+    let user = {
+      email: email,
+      name: name,
+      password: password,
+      password2: secondPassword
```

Comment:
We changed this field name to "confirmPassword" on the backend.

Reply:
ok, i'll make a note to update this once i pull in the backend changes